src/main/java/TerminalUI.java

```diff
@@ -0,0 +1,32 @@
+import java.util.List;
+
+public class TerminalUI {
+    private static final String SEPARATOR_LINE = "________________________________________________________________________________";
+
+    public static void printSepLine() {
+        System.out.println(SEPARATOR_LINE);
+    }
+
+    public static String wrapWithSepLine(String input) {
```

Comment:
Maybe the method name can be rennamed to something along the lines of "wrapWithSeparator" to make it slightly more descriptive